OpenRA.Mods.Common/Lint/CheckTargetHealthRadius.cs

```diff
@@ -32,6 +37,9 @@ public void Run(Action<string> emitError, Action<string> emitWarning, Ruleset ru
 				if (!targetable.Any())
 					continue;
 
+				var blockerTraits = actorInfo.Value.TraitInfos<BlocksProjectilesInfo>().ToList();
+				var gate = actorInfo.Value.TraitInfoOrDefault<GateInfo>();
```

Comment:
Hardcoding `GateInfo` kind of sucks.  This would be a good time to introduce an `IBlocksProjectilesInfo` interface, and then have `GateInfo` define `WDist IBlocksProjectilesInfo.BlockingHeight { get { return BlocksProjectilesHeight; } }`.